decidim-core/app/cells/decidim/content_blocks/organization_metrics_cell.rb

```diff
@@ -2,10 +2,18 @@
 
 module Decidim
   module ContentBlocks
-    class MetricsCell < Decidim::ViewModel
+    class OrganizationMetricsCell < Decidim::ContentBlocks::ParticipatorySpaceMetricsCell
+      private
+
       def metrics
         @metrics ||= MetricChartsPresenter.new(organization: current_organization)
       end
+
+      def scope
+        "home"
+      end
+
+      def display_not_highlighted_metrics = true
```

Comment:
I prefer not to use an endless method here as it's not being used above in this file